README.md

```diff
@@ -107,8 +107,24 @@ docker run -p 8000:8000 --env-file .env todo-app:prod
 with bind mount
 docker run --mount type=bind,src=C:/Users/Rajesh/Devops_project/DevOps-Course-Starter/todo_app,dst=/app/src/todo_app -p 8000:8000 --env-file .env  todo-app:prod
 
+     
+#########
+
 Module 6 C4 Diagram
 The diagram to be openend using : https://app.diagrams.net/
 This app was used to create the diagrams
 
-There are 3 separate tabs for Context, container and component diagram
\ No newline at end of file
+There are 3 separate tabs for Context, container and component diagram
+
+#############
+Exercise-7 commands
+#docker build --target test --tag my-test-image .
+
+#docker run my-test-image test/test_viewmodel.py
+#docker run my-test-image test/test_trello_mock.py
+
+#with parameters
+#docker run  -e TOKEN -e KEY my-test-image test/test_selenium.py
+#with env file
+#docker run --env-file .env my-test-image test/test_selenium.py
+# Testing commands for paths-ignore
```

Comment:
Not sure why you're starting each line with a hash.

This is a markdown file (.md). You can see the result of the formatting here: https://github.com/rajeshdewan/-DevOps-Course-Starter/blob/f7d9c2bdbed63a91ac5c99e7b87603248f30cba6/README.md 

Or you can preview it in VS Code.

A hash means a large header.

If you want code snippets you probably want code blocks with triple backticks.
````
```
some code here
```
````

Reply:
Review comments incorporated